.travis.yml

```diff
@@ -4,6 +4,9 @@ python:
 addons:
   postgresql: "9.4"
 
+env:
+    - repo_token: 6ggWHxHTG2JvepQ98oe4baMzh7EWPOtDM
```

Comment:
@tomuhenry Add it to the environment variables on travis, it should not be visible here. Thanks